helpers.js

```diff
@@ -492,6 +492,17 @@ const getArmletClient = (ethAddress, password, clientToolName = 'truffle') => {
  * @param {Object} config - truffle configuration object.
  */
 async function analyze(config) {
+
+    try {
+        const projectConfig = require([config.working_directory, 'truffle-security.json'].join ('/'));
+        // command line options should overwrite project-level config
+        config = Object.assign(projectConfig, config);
+    } catch (ex) {
+        if (config.debug > 1) {
+            console.log("No truffle security configuration file found");
```

Comment:
`config.logger.error` would probably be more appropriate than `console.log` here.

We can also update the error message to include the possibility of an invalidly formatted file, as that would raise an exception as well.